specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/iotDefenderSettings.json

```diff
@@ -0,0 +1,445 @@
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "IoT Defender settings deleted"
+          },
+          "204": {
+            "description": "IoT Defender settings does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/iotDefenderSettings/default/packageDownloads": {
+      "post": {
+        "x-ms-examples": {
+          "Package downloads": {
+            "$ref": "./examples/IotDefenderSettings/PackageDownloads.json"
+          }
+        },
+        "tags": [
+          "IoT Defender Settings"
+        ],
+        "operationId": "IotDefenderSettings_PackageDownloads",
+        "description": "Information about downloadable packages",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Package downloads",
+            "schema": {
+              "$ref": "#/definitions/PackageDownloads"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IotDefenderSettingsList": {
+      "description": "List of IoT Defender settings",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List data",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IotDefenderSettingsModel"
+          }
+        }
+      }
+    },
+    "IotDefenderSettingsModel": {
+      "type": "object",
+      "description": "IoT Defender settings",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "IoT Defender settings properties",
+          "$ref": "#/definitions/IotDefenderSettingsProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "IotDefenderSettingsProperties": {
+      "type": "object",
+      "description": "IoT Defender settings properties",
+      "properties": {}
```

Comment:
No properties for this object?

Reply:
Planned to be added in the future, but not as part of this preview API version.